.storybook/preview.tsx

```
@@ -0,0 +1,150 @@
+          {
+            // Disable the rule that requires form elements to have labels
+            // as we handle this with our custom FormField component
+            id: "label",
+            selector: ":not(.form-field) > :is(input, select, textarea)",
+          },
+        ],
+      },
+    },
+    viewport: {
+      viewports: INITIAL_VIEWPORTS,
+    },
```

Biome (1.9.4)

[error] This property value named viewport is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named viewport


(lint/suspicious/noDuplicateObjectKeys)